cmake.groovy

```diff
@@ -4,7 +4,7 @@ import com.gargoylesoftware.htmlunit.html.*;
 import net.sf.json.*
 import com.gargoylesoftware.htmlunit.WebClient
 
-def baseUrl = 'http://www.cmake.org/files/'
+def baseUrl = 'https://www.cmake.org/files/'
```

Comment:
The canonical base URL appears to be `https://cmake.org` (i.e. without `www.` prefix) — using that prefix (or `http://`) gets 301-redirected, which Jenkins currently [can't handle](https://issues.jenkins-ci.org/browse/JENKINS-23507).

```
$ curl -I -L https://www.cmake.org/files/v3.3/cmake-3.3.2-win32-x86.zip
HTTP/1.1 301 Moved Permanently
Date: Thu, 05 Nov 2015 18:04:00 GMT
Server: Apache
Location: https://cmake.org/files/v3.3/cmake-3.3.2-win32-x86.zip
Vary: Accept-Encoding
Connection: close
Content-Type: text/html; charset=iso-8859-1

HTTP/1.1 200 OK
Date: Thu, 05 Nov 2015 18:04:01 GMT
Server: Apache
Last-Modified: Thu, 17 Sep 2015 18:35:27 GMT
ETag: "2881e75-fa5736-51ff5adce99c0"
Accept-Ranges: bytes
Content-Length: 16406326
Connection: close
Content-Type: application/zip
```